### PR TITLE
[WIPTEST] style3 class has been (or will soon be) removed from tables

### PR DESCRIPTION
**Pull request by seandst**
There are some review comments sprinked throughout the code, look for "REVIEW:"

Almost universally (the one possible exception needs review), the "style3" class wasn't necessary to locate data tables, since the next locatable table (whether the descendance is '//' or '/') is the table we want.

There were also a lot of instances where the xpath looked like `//div//table` when table was a direct descendant of the located div, so only a single slash is needed. I change those instances when I was sure that this is the case, so places where we still see `//div//table` are probably correct. I wouldn't mind a second look.

### CODE REVIEW

**Comment by seandst** [comment]
The main goal here was to fix the zone_crud smoke tests, which it does, and I assume we'll see that reflected in early PRT results. I'll put together a proper py.test config for PRT soon if nobody beats me to it.

File: cfme/automate/explorer.py [hunk 1]
```diff
@@ -613,7 +613,10 @@ class InstanceFieldsRow(pretty.Pretty):
     Args:
         row_id: Sequential id of the row (begins with 0)
     """
-    table = Table("//div[@id='form_div']//table[@class='style3']")
+    # REVIEW: This locator seems wrong. The instance field table in 53z and
+    # upstream appears to be a SplitTable under div#inst_grid_div
+    # On an instance's page, div#form_div doesn't seem to exist.
+    table = Table("//div[@id='form_div']//table")
```
Comment (seandst) [inline cfme/automate/explorer.py]: Needs review.

**Comment by dajoRH** [comment]
Lint report for commit 857a93d94d69a3ae14fcdfad1cb663176188f322:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/automate/explorer.py (hunk 1 above)
Comment (seandst) [inline cfme/automate/explorer.py]: @mfalesni, do you have any insight here?

File: cfme/configure/tasks.py [hunk 2]
```diff
@@ -44,8 +44,10 @@
     ]
 )
 
+# REVIEW: I expect masterToggle is already wrapped up in mfalesni's
+# Input locator PR
 tasks_table = CheckboxTable(
-    table_locator='//div[@id="records_div"]/table[@class="style3"]',
+    table_locator='//div[@id="records_div"]/table',
```
Comment (seandst) [inline cfme/configure/tasks.py]: Doesn't look like it...I'll probably just fix it.

File: cfme/infrastructure/host.py [hunk 3]
```diff
@@ -67,6 +67,7 @@
     }
 )
 
+# REVIEW: Need to find one of these, would love to have a div#id parent
 drift_table = CheckboxTable("//table[@class='style3']")
```
Comment (seandst) [inline cfme/infrastructure/host.py]: @jkrocil (or anyone) Do you have an appliance handy that already has drift results, so we can fix this locator up? Even better would be two appliances, a 53z and a 54z or newer.

File: cfme/web_ui/__init__.py [hunk 4]
```diff
@@ -977,10 +977,11 @@ def table_in_object(table_title):
         table_title: Text in `p` element preceeding the table
     Returns: XPath locator for the desired table.
     """
+    # REVIEW: What uses this? I wasn't able to test it.
     # Description     paragraph with the text       following element    which is the table
     return (
         "//p[@class='legend' and normalize-space(text())='{}']"
-        "/following-sibling::*[1][@class='style3']").format(table_title)
+        "/following-sibling::table").format(table_title)
```
Comment (seandst) [inline cfme/web_ui/__init__.py]: Need some help finding one of this in the wild. I'm particularly concerned that `following-sibling[0]::*` might also be a table, though to comment above indicated that it's a paragraph, and this should work fine.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please?
*CFME QE Bot*

File: cfme/infrastructure/host.py (hunk 3 above)
Comment (psav) [inline cfme/infrastructure/host.py]: Yeh this one looks a little dangerous

File: cfme/web_ui/__init__.py (hunk 4 above)
Comment (psav) [inline cfme/web_ui/__init__.py]: https://github.com/RedHatQE/cfme_tests/commit/2dbf1093012ac29f6fb54469bd62fe0fb019b4d2#diff-7f7fa4e53ebd99803657de18407cebfeR150 There are a few lines in that file which reference it should give you a pretty good indication of where to find them.

File: cfme/automate/explorer.py (hunk 1 above)
Comment (mfalesni) [inline cfme/automate/explorer.py]: @seandst I don't remember but I can take a look at it.

Comment (mfalesni) [inline cfme/automate/explorer.py]: @seandst I checked in instances, that the locator that works for the fields table is

```
//div[@id='form_div']/table[preceding-sibling::*[.='Fields']]
```

**Comment by seandst** [comment]
Going to hold off on rebasing until we're done the Input locator mess, since that's almost complete.

File: cfme/infrastructure/host.py (hunk 3 above)
Comment (jkrocil) [inline cfme/infrastructure/host.py]: This works both in 5.3.z and 5.4.0:
`"//div[@id='main_div']/table"`

File: cfme/configure/tasks.py (hunk 2 above)
Comment (jkrocil) [inline cfme/configure/tasks.py]: Works in 5.3.z and 5.4.0 (locates exactly one table).

File: cfme/web_ui/__init__.py (hunk 4 above)
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: I have already fixed this function and it is already merged.

**Comment by seandst** [comment]
Input locator is still ongoing, and meanwhile we're taking this out as they come up, so this PR isn't needed anymore.

**Pull request closed without merging**